shell/platform/darwin/ios/framework/Source/vsync_waiter_ios.mm

```diff
@@ -142,6 +143,20 @@ - (CADisplayLink*)getDisplayLink {
   return display_link_.get();
 }
 
+- (void)setMaxRefreshRate:(double)refreshRate {
```

Comment:
Might be better to move this method to line 79 so the diff history from `setMaxRefreshRateIfEnabled` isn't totally lost.